src/Randomizer.SMZ3.Tracking/AutoTracking/EmulatorAction.cs

```diff
@@ -54,6 +54,16 @@ public class EmulatorAction
         /// </summary>
         public EmulatorMemoryData? CurrentData { get; protected set; }
 
+        /// <summary>
+        /// How frequently this should be ran
+        /// </summary>
+        public double FrequencySeconds = 0;
```

Comment:
This comment doesn't really tell me much. This is the number of seconds between a check, right? Between the name Frequency and the property being a `double` it made me wonder at first glance if this wasn't a "number of times per second" thing instead. Maybe worth clarifying?